neuralpp/test/quick_tests/symbolic/expression_test.py

```diff
@@ -266,3 +301,9 @@ def test_sympy_z3_conversion():
     add2 = Z3Expression.new_function_application(real_add_func, [constant_one_third2, two_third])
     assert add2.arguments[1] == add0
     assert add2 == add1
+
+
+def test_sympy_neg_weird():
+    x = SymPyVariable(sympy.symbols("x"), int)
+    neg_x = -x
+    assert neg_x.function.value == operator.mul  # because it is actually "-1 * x"
```

Comment:
I don't know if that's a desired behavior. Not very straight-forward to an end-user.